### ISSUE TITLE
S kernel bugfix

### ISSUE THREAD

**Issue by kasra-hosseini** [seismology/mc_kernel#70]
@EoghanTotten reported the following fixes: (@harriet-godwin and @sstaehler, would you please confirm these changes?)

https://github.com/seismology/mc_kernel/blob/9c534a9bf6346ffaf29453cfc5fb85e6f048d0bb/src/readfields.f90#L1294-L1296

should be changed to:

```
load_fw_points(:,:,ipoint) = rotate_symm_tensor_voigt_src_to_xyz( &
                                          load_fw_points(:,:,ipoint),        &
                                          rotmesh_phi(ipoint), this%ndumps    )
```

and

https://github.com/seismology/mc_kernel/blob/9c534a9bf6346ffaf29453cfc5fb85e6f048d0bb/src/readfields.f90#L1456-L1457

to:
```
load_bw_points(:,:,ipoint) = rotate_symm_tensor_voigt_src_to_xyz(load_bw_points(:,:,ipoint), &
                                          rotmesh_phi(ipoint), this%ndumps)
```

**Comment by sstaehler** [comment]
Yes, sounds reasonable to me. @harriet-godwin, do you agree?

**Comment by harriet-godwin** [comment]
Yes this is the rotation fix that I found a while ago and is correct

**Comment by esleokan** [comment]
I tested a simple SKS kernel at 100º distance in radial component. Now the S kernel shows the expected negative sensitivity in the first Fresnel zone. 
![SKS_100](https://user-images.githubusercontent.com/7670678/55823277-aef69000-5b01-11e9-8683-5d97904da1d2.png)
